packages/@uppy/aws-s3/src/index.js

```diff
@@ -135,13 +135,21 @@ module.exports = class AwsS3 extends Plugin {
     const valid = typeof params === 'object' && params &&
       typeof params.url === 'string' &&
       (typeof params.fields === 'object' || params.fields == null) &&
```

Comment:
woops my bad, applying those caused this line to become wrong.
```suggestion
      (typeof params.fields === 'object' || params.fields == null)
```